www/shared/components/cookie-banner/CookieBanner.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+const mockCookieConsents = [];
+const mockAdd = jest.fn().mockImplementation((cookie) => mockCookieConsents.push(cookie));
+
+jest.doMock('@enzsft/react-cookie-consents',
```

Comment:
Yes, I'll definitely change to `.mock`. `.doMock` is already at the top of the code block and we don't need to avoid hoisting so `.mock` is the way to go, thanks ☺️